utils/run_in_ga.sh

```diff
@@ -34,7 +34,7 @@ utils/check.sh -n /opt/daos/bin/dmg
 echo ::endgroup::
 
 echo ::group::Test incremental debug build with test target
-$SCONS --jobs 10 test
+$SCONS --jobs 10 test client
```

Comment:
if test implies client then why add it here?

Reply:
It tests that scons test client actually builds test.   With the bug, it wouldn't have done so.